Tensile/Common.py

```diff
@@ -1752,45 +1795,54 @@ def assignGlobalParameters( config ):
   globalParameters["ROCmPath"] = "/opt/rocm"
   if "ROCM_PATH" in os.environ:
     globalParameters["ROCmPath"] = os.environ.get("ROCM_PATH")
-  if "TENSILE_ROCM_PATH" in os.environ:
```

Comment:
Is anyone still using the TENSILE_ROCM_PATH for anything?

There appear to be other uses of TENSILE_ROCM_*  symbols below.

Reply:
Good catch.  Sorry about that, it shouldn't have been removed I'll add it back. 
Not sure why you want to potentially mix two ROCMs but that is fine by windows build.